src/libopensc/dir.c

```diff
@@ -60,7 +60,7 @@ static const struct sc_asn1_entry c_asn1_dir[] = {
 };
 
 
-static int
+int
 parse_dir_record(sc_card_t *card, u8 ** buf, size_t *buflen, int rec_nr)
```

Comment:
if it is not needed anymore, please, keep this static.

Reply:
done